src-tauri/src/cli.rs

```diff
@@ -45,6 +45,31 @@ pub struct AddLink {
     pub description: Option<String>,
 }
 
+pub fn process_help() {
```

Comment:
But can it be automatically generated? I think, [clap](https://docs.rs/clap/latest/clap/) crate allows to derive the help message from actual command definitions.

Reply:
noted, let me check. It's just because we use tauri's cli config, which is also using clap, but when I change the help in the configuration, it didn't change.

Reply:
@gwendalF could you look into this?